src/app.tsx

```
@@ -0,0 +1,9 @@
+import { ToastProvider } from "./components/ui/toast"
+
+function App() {
```

ESLint

[error] 'App' is defined but never used.

(@typescript-eslint/no-unused-vars)